lib/authentication/view/sign_up_view.dart

```diff
@@ -187,7 +123,12 @@ class _SignUpViewState extends State<SignUpView> {
     return AppForm(
       title: S.of(context).actionSignUp,
       items: _buildFormItems(),
-      trailing: _dropdownTree(context) + [_privacyPolicy()],
+      trailing: <Widget>[
+            DropdownTree(
+              controller: dropdownController,
+            )
+          ] +
+          [_privacyPolicy()],
```

Comment:
Just make a simple list with both items, no point in concatenating lists here. Concatenation was needed before because _dropdownTree returned a list of widgets.
```suggestion
      trailing: [
            DropdownTree(
              controller: dropdownController,
            ),
            _privacyPolicy(),
      ],
```